src/mlir/ast-utils.h

```diff
@@ -45,36 +45,37 @@ namespace mlir::verona::ASTInterface
   enum NodeKind : NodeType
   {
     None = 0,
-    Module = peg::str2tag("module"),
-    ClassDef = peg::str2tag("classdef"),
-    Function = peg::str2tag("function"),
-    FuncName = peg::str2tag("funcname"),
-    Sig = peg::str2tag("sig"),
-    Block = peg::str2tag("block"),
-    OfType = peg::str2tag("oftype"),
-    Constraints = peg::str2tag("constraints"),
-    Constraint = peg::str2tag("constraint"),
-    Type = peg::str2tag("type"),
-    TypeRef = peg::str2tag("type_ref"),
-    TypeBody = peg::str2tag("typebody"),
-    Params = peg::str2tag("params"),
-    NamedParam = peg::str2tag("namedparam"),
-    ID = peg::str2tag("id"),
-    Seq = peg::str2tag("seq"),
-    Assign = peg::str2tag("assign"),
-    Let = peg::str2tag("let"),
-    Call = peg::str2tag("call"),
-    Args = peg::str2tag("args"),
-    Integer = peg::str2tag("int"),
-    Local = peg::str2tag("local"),
-    Localref = peg::str2tag("localref"),
-    Float = peg::str2tag("float"),
-    Condition = peg::str2tag("cond"),
-    If = peg::str2tag("if"),
-    Else = peg::str2tag("else"),
-    While = peg::str2tag("while"),
-    Continue = peg::str2tag("continue"),
-    Break = peg::str2tag("break"),
+    Module = peg::str2tag("module"), // 73005690
```

Comment:
What are these numbers?

Reply:
The integral value of calling `peg::str2tag("nodename")`, and consequently, the integral value of the enum item.

Perhaps I could make the comment `// = NNN` to make it clearer?